release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -154,7 +154,7 @@ Added support for generating client resource methods in the client generation co
 
 ##### OpenAPI Tool
 ###### OpenAPI to Ballerina Schema Generation
```

Comment:
IMO we don't need to have this `OpenAPI to Ballerina Schema Generation` subheading. we can imply that in the improvement description itself.

Reply:
+1 , will change and remove